.travis.yml

```diff
@@ -7,3 +7,10 @@ cache:
   - $HOME/.gradle
 jdk:
   - oraclejdk8
+env:
+  matrix:
+    - CI_MODE=Released
+  matrix:
+    fast_finish: true
+    allow_failures:
+      - env: "CI_MODE=Nightly"
```

Comment:
copied this from angular -

https://github.com/angular/angular/blob/master/.travis.yml#L56